unix_integration/src/tool.rs

```diff
@@ -58,37 +61,99 @@ async fn main() -> ExitCode {
             return ExitCode::FAILURE
         };
 
-            let password = match rpassword::prompt_password("Enter Unix password: ") {
```

Comment:
The password prompt here will change for auth test, since we rely on the id provider to specify the prompt now (it is currently defaulting to "Password: ").